hibernate-core/src/main/java/org/hibernate/cfg/beanvalidation/BeanValidationIntegrator.java

```diff
@@ -32,7 +32,7 @@
 
 	public static final String APPLY_CONSTRAINTS = "hibernate.validator.apply_to_ddl";
 
-	public static final String BV_CHECK_CLASS = "javax.validation.Validation";
+	public static final String BV_CHECK_CLASS = "javax.validation.ConstraintViolation";
 
 	public static final String MODE_PROPERTY = "javax.persistence.validation.mode";
 
```

Comment:
Saw the original `javax.validation.Validation` comes from the following gradle dependency:
```
validation:     "javax.validation:validation-api:${validationApiVersion}",
```
Do we need to update it or delete it?

Reply:
We could remove it, but there's no benefit from removing it as it is a transitive dependency: better keep the explicit version control.

Reply:
I don't understand your question? The new class comes from the exact same dependency, there's nothing changed here.

Reply:
It is my fault, I didn't know the library contains `javax.validation.ConstraintViolation` as well.